source/Dig/base.util.registry.bmx

```diff
@@ -163,12 +177,26 @@ Function GetRegistry:TRegistry()
 	Return TRegistry.GetInstance()
 End Function
 
-Function GetDataFromRegistry:TData(key:Object, defaultNameOrObject:Object = Null)
-	Return TData( GetRegistry().Get(key, defaultNameOrObject, "data") )
+Function GetDataFromRegistry:TData(key:Object)
+	Return TData( GetRegistry().Get(key) )
 End Function
 
-Function GetStringFromRegistry:String(key:Object, defaultNameOrObject:Object = Null)
-	Return String( GetRegistry().Get(key, defaultNameOrObject, "string") )
+Function GetDataFromRegistry:TData(key:Object, defaultNameOrObject:Object)
+	Global lsKey:TLowerString = TLowerString.Create("data")
```

Comment:
Interessehalber: was ist die Auswirkung von Global für die Variable im Funktionsaufruf. Insb. weil es unter demselben Namen in unterschiedlichen Funktionen globale Variablen gibt.
Werden die dann explizit für diese Funktion einmalig definiert?

Reply:
Global in Funktion passt. Mir war diese Syntax nur noch nie aufgefallen.